mpop_kiosk/main.qml

```diff
@@ -71,6 +71,16 @@ ApplicationWindow {
         mainStackLayout.currentIndex = mainStackLayout.index_SCREENSAVER;
         userProfile.userId = -1;
         window.userProfile.clearAllCallbackIds();
+
+        window.userProfile.freeTag(function(err){
```

Comment:
We'll need to call this only if a given set of conditions are met. For example, we need to be in the exit kiosk. Also, we should only do it once we done answering the final three (3) questions.